tests/store/_unity_catalog/model_registry/test_unity_catalog_rest_store.py

```diff
@@ -0,0 +1,219 @@
+from itertools import combinations
+
+import json
+import pytest
+from unittest import mock
+import functools
+
+from mlflow.entities.model_registry import RegisteredModelTag
+from mlflow.exceptions import MlflowException
+from mlflow.protos.databricks_uc_registry_messages_pb2 import (
+    CreateRegisteredModelRequest,
+    UpdateRegisteredModelRequest,
+    DeleteRegisteredModelRequest,
+    GetRegisteredModelRequest,
+    SearchRegisteredModelsRequest,
+)
+from mlflow.store._unity_catalog.registry.rest_store import UcModelRegistryStore
+from mlflow.utils.proto_json_utils import message_to_json
+from mlflow.utils.rest_utils import MlflowHostCreds
+
+
+@pytest.fixture(scope="module", autouse=True)
+def request_fixture():
+    with mock.patch("requests.request") as request_mock:
+        response = mock.MagicMock()
+        response.status_code = 200
+        response.text = "{}"
+        request_mock.return_value = response
+        yield request_mock
+
+
+def mock_http_request(f):
+    @functools.wraps(f)
+    @mock.patch(
+        "mlflow.utils.rest_utils.http_request",
+        return_value=mock.MagicMock(status_code=200, text="{}"),
+    )
+    def wrapper(*args, **kwargs):
+        return f(*args, **kwargs)
+
+    return wrapper
+
+
+def mock_multiple_http_requests(f):
+    @functools.wraps(f)
+    @mock.patch(
+        "mlflow.utils.rest_utils.http_request",
+        side_effect=[
+            mock.MagicMock(status_code=403, text='{"error_code": "ENDPOINT_NOT_FOUND"}'),
+            mock.MagicMock(status_code=200, text="{}"),
+        ],
+    )
+    def wrapper(*args, **kwargs):
+        return f(*args, **kwargs)
+
+    return wrapper
+
+
+def host_creds():
+    return MlflowHostCreds("https://hello")
+
+
+@pytest.fixture
+def store():
+    yield UcModelRegistryStore(host_creds)
```

Comment:
```suggestion
    return UcModelRegistryStore(host_creds)
```

because we don't perform any teardown operations.